projects/common/c5soc/c5soc_system_qsys.tcl

```diff
@@ -156,86 +156,74 @@ add_connection sys_clk.clk_reset sys_int_mem.reset1
+set_instance_parameter_value pixel_clk_pll_reconfig {ENABLE_MIF} {0}
+add_connection pixel_clk_pll.reconfig_from_pll pixel_clk_pll_reconfig.reconfig_from_pll
+set_connection_parameter_value pixel_clk_pll.reconfig_from_pll/pixel_clk_pll_reconfig.reconfig_from_pll endPortLSB {0}
+set_connection_parameter_value pixel_clk_pll.reconfig_from_pll/pixel_clk_pll_reconfig.reconfig_from_pll startPortLSB {0}
+set_connection_parameter_value pixel_clk_pll.reconfig_from_pll/pixel_clk_pll_reconfig.reconfig_from_pll width {0}
+add_connection pixel_clk_pll.reconfig_to_pll pixel_clk_pll_reconfig.reconfig_to_pll
+set_connection_parameter_value pixel_clk_pll.reconfig_to_pll/pixel_clk_pll_reconfig.reconfig_to_pll endPortLSB {0}
+set_connection_parameter_value pixel_clk_pll.reconfig_to_pll/pixel_clk_pll_reconfig.reconfig_to_pll startPortLSB {0}
+set_connection_parameter_value pixel_clk_pll.reconfig_to_pll/pixel_clk_pll_reconfig.reconfig_to_pll width {0}
+
+#display (video-dmac)
+
+add_instance video_dmac axi_dmac
+set_instance_parameter_value video_dmac {AUTO_ASYNC_CLK} {1}
+set_instance_parameter_value video_dmac {AXI_SLICE_DEST} {0}
+set_instance_parameter_value video_dmac {AXI_SLICE_SRC} {0}
+set_instance_parameter_value video_dmac {CYCLIC} {1}
+set_instance_parameter_value video_dmac {HAS_AXIS_TLAST} {1}
+set_instance_parameter_value video_dmac {DMA_2D_TRANSFER} {1}
+set_instance_parameter_value video_dmac {DMA_DATA_WIDTH_DEST} {64}
+set_instance_parameter_value video_dmac {DMA_DATA_WIDTH_SRC} {64}
+set_instance_parameter_value video_dmac {DMA_LENGTH_WIDTH} {24}
+set_instance_parameter_value video_dmac {DMA_TYPE_DEST} {1}
+set_instance_parameter_value video_dmac {DMA_TYPE_SRC} {0}
+set_instance_parameter_value video_dmac {FIFO_SIZE} {8}
+set_instance_parameter_value video_dmac {ID} {0}
+set_instance_parameter_value video_dmac {SYNC_TRANSFER_START} {0}
+add_connection video_dmac.m_axis vga_out.vdma_if axi4stream
+add_connection sys_clk.clk           pixel_clk_pll.refclk
+add_connection sys_clk.clk           pixel_clk_pll_reconfig.mgmt_clk
+add_connection sys_clk.clk           vga_out.s_axi_clock
+add_connection sys_clk.clk           video_dmac.s_axi_clock
+add_connection pixel_clk_pll.outclk1 video_dmac.m_src_axi_clock
+add_connection pixel_clk_pll.outclk1 video_dmac.if_m_axis_aclk
+add_connection pixel_clk_pll.outclk1 sys_hps.f2h_sdram0_clock
+add_connection pixel_clk_pll.outclk1 vga_out.vdma_clock
+add_connection pixel_clk_pll.outclk0 vga_out.reference_clk
+add_connection sys_clk.clk_reset     pixel_clk_pll.reset
+add_connection sys_clk.clk_reset     pixel_clk_pll_reconfig.mgmt_reset
+add_connection sys_clk.clk_reset     vga_out.s_axi_reset
+add_connection sys_clk.clk_reset     video_dmac.m_src_axi_reset
+add_connection sys_clk.clk_reset     video_dmac.s_axi_reset
+add_connection video_dmac.m_src_axi sys_hps.f2h_sdram0_data
+set_connection_parameter_value video_dmac.m_src_axi/sys_hps.f2h_sdram0_data baseAddress {0x0000}
```

Comment:
The connections should be part of the IP instantiation section, where possible.

Reply:
Could you also use ad_dma_interconnect possible: https://github.com/analogdevicesinc/hdl/blob/master/projects/arradio/common/arradio_qsys.tcl#L144

Reply:
Could you also use ad_dma_interconnect possible: https://github.com/analogdevicesinc/hdl/blob/master/projects/arradio/common/arradio_qsys.tcl#L144